packages/palette/src/elements/Input/Input.test.tsx

```diff
@@ -49,98 +49,52 @@ describe("Input", () => {
 
 describe("computeBorderColor", () => {
   it("defaults to returning black 10", () => {
-    const props = {
-      disabled: null,
-      error: null,
-    }
-    const color = computeBorderColor(props)
+    const color = computeBorderColor(null, null)
     expect(color).toEqual("black10")
   })
 
   it("returns black10 when disabled", () => {
-    const props = {
-      disabled: true,
-      error: false,
-    }
-    const color = computeBorderColor(props)
+    const color = computeBorderColor(true, false)
```

Comment:
Meaning, what is "true", what is "false"? Objects with keys also allow for order independence.